miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -471,31 +471,63 @@ Box.ui = {
       {
         name: "Print Barcodes by Position",
         action: function () {
-          // Ignore items; it's a mess of different object types
-          Utils.printSelectDialog(function (printer, copies) {
-            var input =
-              items.length == 0
-                ? Box.boxJSON.items.map(function (i) {
-                    return i.coordinates;
-                  })
-                : positionStrings;
-            Utils.ajaxWithDialog(
-              "Printing",
-              "POST",
-              Urls.rest.printers.printBoxPositions(printer),
-              {
-                boxId: Box.boxId,
-                positions: input,
-                copies: copies,
+          var dialog = jQuery("#dialogDialog");
+          jQuery("#dialogInfoBelow").html("");
+          jQuery("#dialogInfoAbove").html(
+            "<p>Please select how you would like to sort the barcodes for printing.</p>"
+          );
+          jQuery("#dialogVisual").html(
+            '<p><input type="radio" name="sortOrder" id="sortByRow" value="row" checked/><label for="sortByRow"> Sort by Row</label> ' +
+              '<input type="radio" name="sortOrder" id="sortByColumn" value="column"/><label for="sortByColumn"> Sort by Column</label></p>'
+          );
+          dialog.dialog({
```

Comment:
Use the dialog functions from `lims.js`, which take care of most of the setup you've done here, offer additional features, and keep things consistent